core/commands/log_graph.py

```diff
@@ -55,9 +56,73 @@ def run_async(self):
             return
         line = lines[0]
 
-        commit_hash = line.strip(" |*")[:7]
+        commit_hash = line.strip(" /_\|*")[:7]
         if self.action == "checkout":
             self.checkout_ref(commit_hash)
             util.view.refresh_gitsavvy(self.view)
         elif self.action == "view":
             self.view.window().run_command("gs_show_commit", {"commit_hash": commit_hash})
+
+class GsLogGraphMoreInfoCommand(TextCommand, GitCommand):
+
+    """
+    Show all info about a commit in a quick panel
+    """
+
+    def run(self, edit):
+        selections = self.view.sel()
+        if len(selections) != 1:
+            return
+
+        lines = util.view.get_lines_from_regions(self.view, selections)
+        if not lines:
+            return
+        line = lines[0]
+
+        commit_hash = line.strip(" /_\|*")[:7]
+        if (len(commit_hash) <= 3) :
```

Comment:
The length condition is not really a good reason. It was a fix because  sometimes i had commit_hash = '  ' # two spaces
what should i say?